hack/do-janitor/do-janitor.go

```diff
@@ -0,0 +1,146 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"context"
+	"log"
+	"os"
+	"time"
+
+	"github.com/digitalocean/godo"
+)
+
+const timeToCleanInHours = 12
+
+func main() {
+	log.Println("Starting DO Janitor")
+	token := os.Getenv("DIGITALOCEAN_ACCESS_TOKEN")
+	if token == "" {
+		log.Fatal("missing DO token")
+	}
+
+	ctx := context.Background()
+	client := godo.NewFromToken(token)
+
+	droplets, err := dropletList(ctx, client)
+	if err != nil {
+		log.Fatalf("fail to list droplets: %+v", err.Error())
+	}
+
+	for _, droplet := range droplets {
+		dropletCreated, err := time.Parse(time.RFC3339, droplet.Created)
+		if err != nil {
+			log.Fatalf("failt parsing time: %+v", err.Error())
+		}
+
+		hours := time.Since(dropletCreated).Hours()
+		if hours >= timeToCleanInHours {
+			log.Printf("%s is older than %d hours will terminate\n", droplet.Name, timeToCleanInHours)
+			_, err := client.Droplets.Delete(ctx, droplet.ID)
+			if err != nil {
+				log.Printf("fail to delete droplet %s: %+v\n", droplet.Name, err.Error())
+				continue
+			}
+
+			log.Printf("droplet %s terminated\n", droplet.Name)
+		}
+	}
+
+	lbs, err := lbList(ctx, client)
+	if err != nil {
+		log.Fatalf("fail to failed list LoadBalancer: %+v", err.Error())
+	}
+
+	for _, lb := range lbs {
+		lbCreated, err := time.Parse(time.RFC3339, lb.Created)
+		if err != nil {
+			log.Fatalf("failt parsing time: %+v", err.Error())
+		}
+
+		hours := time.Since(lbCreated).Hours() / 24
```

Comment:
We should not divide with 24 here.

```suggestion
		hours := time.Since(lbCreated).Hours()
```

Reply:
today is not my day :)

Reply:
done